views/pod.html

```diff
@@ -15,7 +15,7 @@
     <dt>Phase</dt>
     <dd>{{resource.status.phase}}</dd>
     <dt>Node</dt>
-    <dd>{{resource.status.host}}</dd>
+    <dd>{{resource.spec.host}}</dd>
```

Comment:
Right.  And node pods aren't polled anymore, the kubelet posts updates, so it's 'selector' -> scheduler -> 'host' -> kubelet -> 'pod/status'

----- Original Message -----
> > @@ -15,7 +15,7 @@
> >      <dt>Phase</dt>
> >      <dd>{{resource.status.phase}}</dd>
> >      <dt>Node</dt>
> > -    <dd>{{resource.status.host}}</dd>
> > +    <dd>{{resource.spec.host}}</dd>
> 
> Ah, so normally, `spec.nodeSelector` -> scheduler -> `spec.host` -> node pod
> polling -> `status.hostIP`/`status.podIP`
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/kubernetes-ui/object-describer/pull/9/files#r29155841